CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+Contributing to ActiveRecord Session Store
```

Comment:
Active Record is always written with space in between them, unless you are referring to the module. (Same goes for other components as well)